sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/regexpExpressions.scala

```diff
@@ -364,7 +364,8 @@ case class RLike(left: Expression, right: Expression) extends StringRegexExpress
 
   override def escape(v: String): String = v
   override def matches(regex: Pattern, str: String): Boolean = regex.matcher(str).find(0)
-  override def toString: String = s"$left RLIKE $right"
```

Comment:
we should also override `prettyName`, similar to average: `override def prettyName: String = getTagValue(FunctionRegistry.FUNC_ALIAS).getOrElse("avg")`